packages/test-app-frontend/src/App/imodel-browser/ITwinBrowser.tsx

```diff
@@ -27,20 +27,38 @@ export function ITwinBrowser(): ReactElement {
     { authorizationClient },
   );
 
+  const navigate = useNavigate();
+
   return (
     <PageLayout.Content padded>
       <div style={{ display: "grid", gap: "var(--iui-size-xl)" }}>
-        <Surface style={{ padding: "var(--iui-size-l)" }}>
-          <div style={{ display: "grid", gap: "var(--iui-size-m)" }}>
-            <Text variant="title">Recent</Text>
-            <Paginator
-              initialUrl={applyUrlPrefix("https://api.bentley.com/itwins/recents?subclass=Project&$top=5")}
-              fetch={fetchITwins}
-            >
-              {(result) => <ITwinTileGrid iTwins={result.iTwins} />}
-            </Paginator>
-          </div>
-        </Surface>
+        <div style={{ display: "grid", grid: "1fr / auto 1fr" }}>
+          <Tile.Wrapper style={{ margin: "var(--iui-size-l)", width: "calc(10 * var(--iui-size-l))" }}>
+            <Tile.ThumbnailArea style={{ height: "calc(5 * var(--iui-size-l))" }}>
```

Comment:
Why inline styling? Export to a style sheet.

Reply:
That's a solid reply. I also didn't know it worked like that for Vite. Thank you for informing me and for laying out the pros and cons in a concise and easy to read manor. Consider this resolved.